src/Build/Construction/ProjectRootElement.cs

```diff
@@ -1649,7 +1649,7 @@ public void ReloadFrom(string path, bool throwIfUnsavedChanges = true, bool? pre
         /// <summary>
         /// Reload the existing project root element from the given <paramref name="reader"/>
         /// A reload operation completely replaces the state of this <see cref="ProjectRootElement"/> object. This operation marks the
-        /// object as dirty (see <see cref="ProjectRootElement.MarkDirty"/> for side effects).
+        /// object as dirty (see <see cref="ProjectRootElementLink.MarkDirty"/> for side effects) .
```

Comment:
This is different class and docs for `ProjectRootElementLink.MarkDirty` is missing. I believe that internal implementation details shall not be mentioned in public methods and consequent generated public documentation.
I recommend moving sentence ` This operation marks the
object as dirty (see <see cref="ProjectRootElementLink.MarkDirty"/> for side effects) .` into method body as regular // comment without using <see

Reply:
Agree, I was scratching my head seeing this one. I wasn't sure if we want to keep the info, but we can definitely move it from xmldoc to somewhere else.

Reply:
I changed from ProjectRootElement.MarkDirty to ProjectRootElementLink.MarkDirty since the ProjectRootElement.MarkDirty is internal ,  ProjectRootElementLink.MarkDirty is public and remote to the ProjectRootElement.MarkDirty.

If the internal implementation details shall not be mentioned in public methods. I think we also need to update https://github.com/dotnet/msbuild/blob/00833d9f8772bc99c48ccbebc013aa0a6d5ee622/src/Build/ObjectModelRemoting/ConstructionObjectLinks/ProjectRootElementLink.cs#L210-L214

Reply:
After some thinking, I would say that the easiest thing to do is to remove non-public member cref (and keep the name for now) to make analyzer happy. But we should think more how to document these public APIs - probably in different issue.

@ladipro, any thoughts?

Reply:
Removed the non public cref